pkg/orchestration/wiring/datadog/plugin.go

```diff
@@ -194,13 +165,10 @@ func (q *QueryParams) generateMessage(notificationProp *meta_orch.Notifications)
 	return msg + messageType
 }
 
-func createAlarmNamesForSmithResource(resourceName voyager.ResourceName, deploymentResourceName smith_v1.ResourceName, alarmType AlarmType) smith_v1.ResourceName {
-	nameList := strings.Join([]string{string(deploymentResourceName), string(alarmType)}, "--")
-	return wiringutil.ServiceInstanceWithPostfixResourceName(resourceName, nameList)
-
-}
-
-func createAlarmNameforDatadog(serviceName string, resourceName string, alarmType string, env string, label string) string {
-	nameList := []string{serviceName, resourceName, alarmType, env, label}
+func createAlarmNameforDatadog(serviceName voyager.ServiceName, envType voyager.EnvType, resourceName voyager.ResourceName, alarmType AlarmType, label voyager.Label) string {
+	nameList := []string{string(serviceName), string(envType), string(resourceName), string(alarmType)}
```

Comment:
Reordered - environment goes after service name because its the next significant thing.